src/Tizen.System.Resource/Resource/Resource.cs

```diff
@@ -0,0 +1,180 @@
+﻿/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.System
+{
+	/// <summary>
+	/// Provides methods to support CPU boosting and CPU inheritance.
+	/// </summary>
+    /// <since_tizen> 9 </since_tizen>
+	public class Resource
+	{
+        /// <summary>
+		/// Set cpu boosting for the target process (pid/tids).
+        /// </summary>
+		/// <since_tizen> 9 </since_tizen>
+		/// <privilege>
+		/// http://tizen.org/privilege/internal/default/partner
+		/// </privilege>
+		/// <param name="pid">The target process pid/tids.</param>
+		/// <param name="level">The cpu boosting level</param>
+		/// <param name="flags">The cpu boosting flag bits</param>
+		/// <param name="timeout_msec">The timeout in milliseconds</param>
+		/// <returns>0 on success</returns>
+		public static int ResourceSetCpuBoosting (ResourcePidInfo pid, CpuBoostingLevel level, CpuBoostingFlag flags, int timeout_msec)
```

Comment:
I think you can use `void` return. the `Returning zero` is not meaningful in this method because an exception will be thrown in error case.